app/smartapi.py

```diff
@@ -1,8 +1,12 @@
 """SmartAPI registry access utility."""
 from functools import cache
 
+import os
 import httpx
 
+# get workflow-runner maturity level
+OPENAPI_SERVER_MATURITY = os.getenv("OPENAPI_SERVER_MATURITY", "development")
```

Comment:
We can pass this in as part of the init. server.py should be able to pass this in easily.

Reply:
I tried to put this after line 16 but I got an error saying that OPENAPI_SERVER_MATURITY was undefined when I tried to use it on line 47.